clickControl/clickControl.py

```diff
@@ -132,272 +190,288 @@ def updateLinks(self, msg, links=[], delays=[], capacities=[], losses=[]):
                 return False
         if len(links) != len(losses):
             if len(losses) != 1 and len(losses) != 0:
-                self.log.error("Click: must specify loss probibility for each link or specify only 1 or 0")
+                self.log.error(
+                    "Click: must specify loss probibility for each link or specify only 1 or 0"
+                )
                 return False
-            
-        c = 0
-        skipDelay = False
-        skipCapacity = False
-        skipLoss = False
+
+        skip_delay = False
+        skip_capacity = False
+        skip_loss = False
         if len(delays) == 0:
-            skipDelay = True
+            skip_delay = True
         if len(capacities) == 0:
-            skipCapacity = True
+            skip_capacity = True
         if len(losses) == 0:
-            skipLoss = True
-            
-        for c in range(len(links)):
-            c_link = links[c]
-            if not skipDelay:
+            skip_loss = True
+
+        for link_number in enumerate(links):
+            c_link = links[link_number]
+            if not skip_delay:
                 c_delay = ""
                 if len(delays) == 1:
                     c_delay = delays[0]
                 else:
-                    c_delay = delays[c]
+                    c_delay = delays[link_number]
 
-                ret = self.updateDelay(msg, link = c_link, delay = c_delay)
+                ret = self.updateDelay(msg, link=c_link, delay=c_delay)
                 if not ret:
                     return False
-                
-            if not skipCapacity:
+
+            if not skip_capacity:
```

Comment:
You changed to camelCase elsewhere and under_score here. Any reason?

Reply:
I missed it - do you have a preference for snake_case (it seems the default notation for python community, pylint defaults the rc file to it).  I can swap the code back to Camel.

Reply:
I prefer snake_case, but either way is OK. We don't have standards here at ISI. :)

Reply:
They say time and pressure makes diamonds.

I'll keep the pylints for snake_case and as time goes on I'll re-write necessary code to prefer python community standards